joern-cli/frontends/gosrc2cpg/src/main/scala/io/joern/gosrc2cpg/GoSrc2Cpg.scala

```diff
@@ -30,7 +30,8 @@ class GoSrc2Cpg extends X2CpgFrontend[Config] {
           GoAstJsonParser.readModFile(Paths.get(modFile)).foreach(x => GoMod.meta = Some(x))
         )
         new AstCreationPass(cpg, astGenResult, config, report).createAndApply()
-        TypeNodePass.withRegisteredTypes(GoGlobal.typesSeen(), cpg).createAndApply()
+        new CallTypeAndSigntureLinkerPass(cpg).createAndApply()
+//        TypeNodePass.withRegisteredTypes(GoGlobal.typesSeen(), cpg).createAndApply()
```

Comment:
Shouldn't this be uncommented again?

Reply:
This is not being used. I wanted to add it only when we start using it.